lms/templates/fields/field_dropdown_account.underscore

```diff
@@ -0,0 +1,42 @@
+<div class="u-field-value field">
+    <% if (editable !== 'never') { %>
+        <% if (title && titleVisible) { %>
+            <label class="u-field-title field-label" for="u-field-select-<%- id %>">
+                <%- title %>
+            </label>
+        <% } else { %>
+            <label class="sr" for="u-field-select-<%- id %>">
+                <%- screenReaderTitle %>
+            </label>
+        <% } %>
+    <% } %>
+
+    <% if (iconName) { %>
+        <span class="u-field-icon icon fa <%- iconName %> fa-fw" aria-hidden="true"></span>
+    <% } %>
+
+    <% if (editable === 'never') { %>
+        <span class="sr"><%- screenReaderTitle %></span>
+        <span class="u-field-value-readonly"></span>
+    <% } else { %>
+        <select name="select" id="u-field-select-<%- id %>" aria-describedby="u-field-help-message-<%- id %>">
+            <% if (showBlankOption) { %>
+                <option value=""></option>
+            <% } %>
+            <% _.each(selectOptions, function(selectOption) { %>
+                <option value="<%- selectOption[0] %>"><%- selectOption[1] %></option>
+            <% }); %>
+        </select>
+        <span class="icon-caret-down" aria-hidden="true"></span>
+        <button class="u-field-value-display">
```

Comment:
This controls cannot be operated using the keyboard using Firefox on MAC or Windows.  Is there a test for this?  Our test environment using firefox.  Any test for keyboard support should fail here.

Not sure why we are not using a standard select menu here.  This reports itself as a button, but its not, the control the user will eventually be using is a select menu.